### PR TITLE
Misc typographical fixes

### PR DESCRIPTION
**Pull request by casswedson**
#### Summary
None

#### Purpose of change
To fix some small typographical errors, note that I am not a native English speaker, (I just happen to have a file with all the descriptions found in the `data/json` folder ~9000 lines, vim's handy spell checker and the internet for references) so apologies for possible obvious mistakes.

### CODE REVIEW

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 1]
```diff
@@ -2767,7 +2767,7 @@
     "type": "terrain",
     "id": "t_plastic_window_with_curtain",
     "name": "Plastic window with a curtain",
-    "description": "A makeshift window with a closed curtain.  comprising a sheet of translucent, rigid plastic secured in a wooden frame.  There are sheets drawn across it to block the light.  It'll do in a pinch.",
+    "description": "A makeshift window with a closed curtain.  Comprising a sheet of translucent, rigid plastic secured in a wooden frame.  There are sheets drawn across it to block the light.  It'll do in a pinch.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:2770]:
```suggestion
    "description": "A makeshift window with a closed curtain.  It's a sheet of translucent, rigid plastic secured in a wooden frame.  There are sheets drawn across it to block the light.  It'll do in a pinch.",
```

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 2]
```diff
@@ -2812,7 +2812,7 @@
     "type": "terrain",
     "id": "t_plastic_window_with_curtain_open_window_closed",
     "name": "Plastic window with a curtain",
-    "description": "A makeshift window with a opened curtain.  comprising a sheet of translucent, rigid plastic secured in a wooden frame, with sheets strung about it to block the light as required.  It'll do in a pinch.",
+    "description": "A makeshift window with a opened curtain.  Comprising a sheet of translucent, rigid plastic secured in a wooden frame, with sheets strung about it to block the light as required.  It'll do in a pinch.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:2815]:
```suggestion
    "description": "A makeshift window with a opened curtain.  It's a sheet of translucent, rigid plastic secured in a wooden frame, with sheets strung about it to block the light as required.  It'll do in a pinch.",
```

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 3]
```diff
@@ -2859,7 +2859,7 @@
     "type": "terrain",
     "id": "t_plastic_window_with_curtain_open",
     "name": "Plastic window With a curtain",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:2861]:
```suggestion
    "name": "Plastic window with open curtains",
```

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 4]
```diff
@@ -2859,7 +2859,7 @@
...
-    "description": "An opened makeshift window with a opened curtain.  comprising a sheet of translucent, rigid plastic secured in a wooden frame, with sheets strung about it to block the light as required.  It'll do in a pinch.",
+    "description": "An opened makeshift window with a opened curtain.  Comprising a sheet of translucent, rigid plastic secured in a wooden frame, with sheets strung about it to block the light as required.  It'll do in a pinch.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:2862]:
```suggestion
    "description": "An open makeshift window with open curtains.  It's a sheet of translucent, rigid plastic secured in a wooden frame, with sheets strung about it to block the light as required.  It'll do in a pinch.",
```

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 5]
```diff
@@ -2986,7 +2986,7 @@
     "type": "terrain",
     "id": "t_reinforced_plastic_window_with_curtain",
     "name": "Reinforced plastic window with a curtain",
-    "description": "A makeshift window with a closed curtain.  comprising three sheets of translucent, rigid plastic secured in a wooden frame.  It'll do in a pinch.",
+    "description": "A makeshift window with a closed curtain.  Comprising three sheets of translucent, rigid plastic secured in a wooden frame.  It'll do in a pinch.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:2989]:
```suggestion
    "description": "A makeshift reinforced window with a closed curtain.  It's three sheets of translucent, rigid plastic secured in a wooden frame.  There are sheets drawn across it to block the light.  It'll do in a pinch.",
```

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 6]
```diff
@@ -3035,7 +3035,7 @@
     "type": "terrain",
     "id": "t_reinforced_plastic_window_with_curtain_open_window_closed",
     "name": "Reinforced plastic window with a curtain",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:3037]:
```suggestion
    "name": "Reinforced plastic window with an open curtain",
```

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 7]
```diff
@@ -3086,7 +3086,7 @@
     "type": "terrain",
     "id": "t_reinforced_plastic_window_with_curtain_open",
     "name": "Reinforced plastic window With a curtain",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:3088]:
```suggestion
    "name": "Reinforced plastic window with an open curtain",
```

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 8]
```diff
@@ -3035,7 +3035,7 @@
     "type": "terrain",
     "id": "t_reinforced_plastic_window_with_curtain_open_window_closed",
     "name": "Reinforced plastic window with a curtain",
-    "description": "A makeshift window with a opened curtain.  comprising three sheets of translucent, rigid plastic secured in a wooden frame.  It'll do in a pinch.",
+    "description": "A makeshift window with a opened curtain.  Comprising three sheets of translucent, rigid plastic secured in a wooden frame.  It'll do in a pinch.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:3038]:
```suggestion
    "description": "A makeshift reinforced window with a opened curtain.  It's three sheets of translucent, rigid plastic secured in a wooden frame, with sheets strung about it to block the light as required.  It'll do in a pinch.",
```

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 9]
```diff
@@ -3086,7 +3086,7 @@
     "type": "terrain",
     "id": "t_reinforced_plastic_window_with_curtain_open",
     "name": "Reinforced plastic window With a curtain",
-    "description": "An open makeshift window with a opened curtain.  comprising three sheets of translucent, rigid plastic secured in a wooden frame.  It'll do in a pinch.",
+    "description": "An open makeshift window with a opened curtain.  Comprising three sheets of translucent, rigid plastic secured in a wooden frame.  It'll do in a pinch.",
```
Comment (actual-nh) [inline data/json/furniture_and_terrain/terrain-windows.json:3089]:
```suggestion
    "description": "An open makeshift reinforced window with a opened curtain.  It's three sheets of translucent, rigid plastic secured in a wooden frame, with sheets strung about it to block the light as required.  It'll do in a pinch.",
```

File: data/json/items/armor/legs_armor.json [hunk 10]
```diff
@@ -479,7 +479,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str_sp": "EOD trousers" },
-    "description": "Thick armored trousers constructed from kevlar and nomex for explosive ordnance disposal.  It is designed to protect against overpressure, fragmentation, impact and heat.",
+    "description": "Thick armored trousers constructed from Kevlar and Nomex for explosive ordnance disposal.  It is designed to protect against overpressure, fragmentation, impact and heat.",
```
Comment (actual-nh) [inline data/json/items/armor/legs_armor.json:482]:
```suggestion
    "description": "Thick armored trousers constructed from Kevlar and Nomex for explosive ordnance disposal.  They are designed to protect against overpressure, fragmentation, impact, and heat.",
```

File: data/json/items/armor/legs_armor.json [hunk 11]
```diff
@@ -499,7 +499,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str_sp": "light EOD trousers" },
-    "description": "Armored trousers constructed from kevlar and nomex designed to protect against overpressure, fragmentation, impact and heat in hostile environments.  It is lighter than normal EOD armor to provide more maneuverability.",
+    "description": "Armored trousers constructed from Kevlar and Nomex designed to protect against overpressure, fragmentation, impact and heat in hostile environments.  It is lighter than normal EOD armor to provide more maneuverability.",
```
Comment (actual-nh) [inline data/json/items/armor/legs_armor.json:502]:
```suggestion
    "description": "Armored trousers constructed from Kevlar and Nomex, designed to protect against overpressure, fragmentation, impact, and heat in hostile environments.  They are lighter than normal EOD armor to provide more maneuverability.",
```

File: data/json/items/armor/torso_armor.json [hunk 12]
```diff
@@ -512,7 +512,7 @@
     "type": "TOOL_ARMOR",
     "category": "armor",
     "name": { "str": "EOD jacket" },
-    "description": "A thick armored jacket constructed from kevlar and nomex for explosive ordnance disposal.  It is designed to protect against overpressure, fragmentation, impact and heat.",
+    "description": "A thick armored jacket constructed from Kevlar and Nomex for explosive ordnance disposal.  It is designed to protect against overpressure, fragmentation, impact and heat.",
```
Comment (actual-nh) [inline data/json/items/armor/torso_armor.json:515]:
```suggestion
    "description": "A thick armored jacket constructed from Kevlar and Nomex for explosive ordnance disposal.  It is designed to protect against overpressure, fragmentation, impact, and heat.",
```

File: data/json/items/armor/torso_armor.json [hunk 13]
```diff
@@ -533,7 +533,7 @@
     "type": "TOOL_ARMOR",
     "category": "armor",
     "name": { "str": "light EOD jacket" },
-    "description": "An armored jacket constructed from kevlar and nomex designed to protect against overpressure, fragmentation, impact and heat in hostile environments.  It is lighter than normal EOD armor to provide more maneuverability and can be worn over ballistic armor.",
+    "description": "An armored jacket constructed from Kevlar and Nomex designed to protect against overpressure, fragmentation, impact and heat in hostile environments.  It is lighter than normal EOD armor to provide more maneuverability and can be worn over ballistic armor.",
```
Comment (actual-nh) [inline data/json/items/armor/torso_armor.json:536]:
```suggestion
    "description": "An armored jacket constructed from Kevlar and Nomex, designed to protect against overpressure, fragmentation, impact, and heat in hostile environments.  It is lighter than normal EOD armor to provide more maneuverability and can be worn over ballistic armor.",
```

File: data/json/items/gun/flintlock.json [hunk 14]
```diff
@@ -118,7 +118,7 @@
     "copy-from": "rifle_flintlock",
     "type": "GUN",
     "name": { "str": "flintlock rifle" },
-    "description": "Also called a Pennsylvania rifle, this long barreled rifled flintlock gun has signficantly increased accuracy at the cost of taking even longer to reload.",
+    "description": "Also called a Pennsylvania rifle, this long barreled rifled flintlock gun has significantly increased accuracy at the cost of taking even longer to reload.",
```
Comment (actual-nh) [inline data/json/items/gun/flintlock.json:121]:
```suggestion
    "description": "Also called a Pennsylvania rifle, this long-barreled rifled flintlock gun has significantly increased accuracy at the cost of taking even longer to reload.",
```

File: data/json/items/gun/shot.json [hunk 15]
```diff
@@ -74,7 +74,7 @@
     "looks_like": "remington_870",
     "type": "GUN",
     "name": { "str": "handmade lever shotgun" },
-    "description": "A short homemade lever-action shotgun with a small internal tube magazine.  While still a primitive pipe and 2x4 design, it is a formiddable shotgun in its own right with room for improvement.",
+    "description": "A short homemade lever-action shotgun with a small internal tube magazine.  While still a primitive pipe and 2x4 design, it is a formidable shotgun in its own right with room for improvement.",
```
Comment (actual-nh) [inline data/json/items/gun/shot.json:77]:
```suggestion
    "description": "A short, homemade lever-action shotgun with a small internal tube magazine.  While still a primitive design - made from a pipe and a 2x4 - it is a formidable shotgun in its own right with room for improvement.",
```
Not sure if "a 2x4" is quite correct according to CDDA practice - will need to check whether it's supposed to be something like "a plank".

File: data/json/items/gun/shot.json [hunk 16]
```diff
@@ -802,7 +802,7 @@
     "copy-from": "shotgun_base",
     "type": "GUN",
     "name": { "str": "M1897 Trench Gun" },
-    "description": "The Winchester 1897 was one of the first commercially successful pump action shotguns.  In its 'trench' configuraton it has become a heavily romanticized American icon of World War 1.  With its barrel shroud, bayonet lug and 17 inch bayonet, this shotgun is undeniably fearsome in appearance.  There aren't any more trenches to clear, so the next zombie infested town will have to suffice.",
+    "description": "The Winchester 1897 was one of the first commercially successful pump action shotguns.  In its 'trench' configuration it has become a heavily romanticized American icon of World War 1.  With its barrel shroud, bayonet lug and 17 inch bayonet, this shotgun is undeniably fearsome in appearance.  There aren't any more trenches to clear, so the next zombie infested town will have to suffice.",
```
Comment (actual-nh) [inline data/json/items/gun/shot.json:805]:
```suggestion
    "description": "The Winchester 1897 was one of the first commercially-successful pump-action shotguns.  In its 'trench' configuration it has become a heavily-romanticized American icon of World War 1.  With its barrel shroud, bayonet lug, and 17 inch bayonet, this shotgun is undeniably fearsome in appearance.  There aren't any more trenches to clear, so the next zombie-infested town will have to suffice.",
```

File: data/json/monsters/jabberwock.json [hunk 17]
```diff
@@ -3,7 +3,7 @@
     "id": "mon_fleshy_shambler",
     "type": "MONSTER",
     "name": { "str": "fleshy shambler" },
-    "description": "An amalgamation of throbbing organs from various creatures have fused together into this lurching, vaguely humanoid shape.  Its myriad roughly formed mouths sussurate in a chorus of sibilant groans and whispers.",
+    "description": "An amalgamation of throbbing organs from various creatures have fused together into this lurching, vaguely humanoid shape.  Its myriad roughly formed mouths susurrate in a chorus of sibilant groans and whispers.",
```
Comment (actual-nh) [inline data/json/monsters/jabberwock.json:6]:
```suggestion
    "description": "An amalgamation of throbbing organs from various creatures have fused together into this lurching, vaguely-humanoid shape.  Its myriad roughly-formed mouths susurrate in a chorus of sibilant groans and whispers.",
```

File: data/json/monsters/zed_acid.json [hunk 18]
```diff
@@ -260,7 +260,7 @@
     "type": "MONSTER",
     "copy-from": "mon_zombie_dog_acidic",
     "name": { "str": "blistered horror" },
-    "description": "A huge canine with multiple large foul looking blisters covering its body.  A corrosive liquid spills from its menancing-looking mouth.",
+    "description": "A huge canine with multiple large foul looking blisters covering its body.  A corrosive liquid spills from its menacing-looking mouth.",
```
Comment (actual-nh) [inline data/json/monsters/zed_acid.json:263]:
```suggestion
    "description": "A huge canine with multiple large, foul-looking blisters covering its body.  A corrosive liquid spills from its menacing-looking mouth.",
```

**Review by actual-nh — COMMENTED** [review]
Some enhancements on your work. I'm not completely certain about the names of various windows in different configurations - as in whether they're supposed to indicate open/closed, etc - but some of them were already different from each other (incorrectly so, using "With" instead of "with").

File: data/json/items/armor/legs_armor.json (hunk 10 above)
Comment (casswedson) [reply data/json/items/armor/legs_armor.json:482]: Fascinating didn't know CDDA preferred serial commas.

File: data/json/items/gun/shot.json (hunk 15 above)
Comment (casswedson) [reply data/json/items/gun/shot.json:77]: #47709 I'll replace it locally

**Comment by actual-nh** [comment]
@BrettDong: This would be Bugfix, JSON, unfortunately String Freeze, and possibly Items/etc. Thanks!

**Pull request merged**